src/components/messenger/list/styles.scss

```diff
@@ -30,12 +30,32 @@ $side-padding: 16px;
       display: flex;
       gap: 24px;
       flex-direction: column;
-      padding-bottom: 16px;
+      padding: $side-padding 0;
 
       animation: conversation-slide-in 600ms ease-in forwards;
 
-      &-conversations-input {
-        padding: 0px $side-padding;
+      &-actions {
+        display: flex;
+        align-items: center;
+        gap: 16px;
+        padding: 0 $side-padding;
+      }
+
+      &-conversations-search {
+        flex: 1;
+
+        // @note: this is here because parent size calculations with flex 1
+        // are a little funky with zUI input right now.
+        input {
+          width: 0;
+        }
```

Comment:
This was a weird one - couldn't exactly work out what was going on here. Will need to dig into `zUI` to work it out, which I'll do another day.